### ISSUE TITLE
Tahaggart Tamahaq (thv)

### ISSUE THREAD

**Issue by cueyayotl** [Tatoeba/tatoeba2#1712]
- Language name: Tahaggart Tamahaq
- Language info: https://en.wikipedia.org/wiki/Tamahaq_language
- ISO 639-3 code: thv
- Public List: https://tatoeba.org/eng/sentences_lists/show/8478
- Icon: https://i.imgur.com/I7a0gNs.png
- Requested by: https://tatoeba.org/eng/user/profile/Aissimahfoud06

[for tatoeba.org]
CALL add_new_language('thv', 8478);

**Comment by mozillakab** [comment]
Hi @cueyayotl 
Aissimahfoud06 says that he can't add the tamahaq sentences to the given list. A matter of rigths?
I also tried to add a sentence of mine as a test but I couldn't.

**Comment by mozillakab** [comment]
@cueyayotl please would you like to confirm that Aissimahfoud06 can add sentences to the list you created. I tried with my account but I can't.

**Comment by trang** [comment]
@mozillakab I just tested and this is a regression, I created an issue for it (#1811).

Don't worry too much about filling up the list. The quantity of sentences in that list will not speed up the addition of the language and any sentence that is not in the list can always be switched to Tahaggart Tamahaq after the language is added.

Please be aware that we are putting on hold the addition of new languages at the moment. It may take a couple of months for this language to be added.

**Comment by mozillakab** [comment]
@trang 
Thanks for help. 

As I said, Aissimahfoud06 is a teacher of Tahaggart Tamahaq in the south of Algeria. He would use Tatoeba with his students et teach and gather more sentences. I will inform him.

**Comment by sabretou** [comment]
@trang Can we add this language and close the issue? I believe it fulfils all our requirements.

**Comment by trang** [comment]
@sabretou Regarding closing issues, language requests are closed only after the language is effectively deployed on Tatoeba. Because even after there's a pull request and even after the source code has been changed, there are some extra commands to run from the server for the language to really be added.
